### PR TITLE
[Fix] Voting rewards

### PR DESCRIPTION
**Pull request by csanti**
### Overview

When users execute un-stake MIR from Gov before claiming voting rewards, the `locked_balance` entry for the finished poll is deleted. Therefore, when the user later tries to withdraw voting rewards, by iterating through `locked_balance` entries, he can not withdraw it.

Even if `locked_balance` entry is deleted, the `VoterInfo` is still there.

### Fix
- Only removed locked_balance entry when: 
   1. The user unstakes and there were no voting rewards for that poll
   2. The user has wihtdrawn the voting rewards 
- Also, we add `poll_id` paramter to claim voting rewards, so that users that lost their `locked_balance` entry, can still claim their rewards, by reading the `VoterInfo` directly.

### CODE REVIEW

File: contracts/mirror_gov/src/staking.rs [hunk 1]
```diff
@@ -299,9 +328,16 @@ fn withdraw_user_voting_rewards<S: Storage>(
...
-) -> u128 {
-    let w_polls: Vec<(Poll, VoterInfo)> =
-        get_withdrawable_polls(storage, token_manager, user_address);
+    poll_id: Option<u64>,
+) -> StdResult<(u128, Vec<u64>)> {
+    let w_polls: Vec<(Poll, VoterInfo)> = match poll_id {
+        Some(poll_id) => {
+            let poll: Poll = poll_read(storage).load(&poll_id.to_be_bytes())?;
+            let voter_info = poll_voter_read(storage, poll_id).load(&user_address.as_slice())?;
+            vec![(poll, voter_info)]
```
Comment (YunSuk-Yeo) [inline contracts/mirror_gov/src/staking.rs:337]: Don't we need to conduct withdrawable check?

Comment (YunSuk-Yeo) [reply contracts/mirror_gov/src/staking.rs:337]: like https://github.com/Mirror-Protocol/mirror-contracts/pull/62/files#diff-b4800aa15fd9828e1b90abbf8f4d797616137eb2e7701b8134759542e6833ad3L337

**Review by YunSuk-Yeo — CHANGES_REQUESTED** [review]

File: contracts/mirror_gov/src/staking.rs (hunk 1 above)
Comment (csanti) [reply contracts/mirror_gov/src/staking.rs:343]: right. Fixed it

**Review by YunSuk-Yeo — COMMENTED** [review]
LGTM

**Pull request merged**